docs_source_files/content/remote_webdriver/remote_webdriver_client.en.md

```diff
@@ -34,7 +34,19 @@ driver.get "http://www.google.com"
 driver.close
   {{< / code-panel >}}
   {{< code-panel language="javascript" >}}
-// We don't have a JavaScript code sample yet -  Help us out and raise a PR  
+const { Builder, Capabilities } = require("selenium-webdriver");
+let chromeCapabilities = Capabilities.chrome();
```

Comment:
Sorry, I don't remember exactly, aren't there `ChromeOptions` in JavaScript?

Reply:
ChromeOptions in javascript nodejs out. Now the value name: chromeOptions is goo:chromeOptions (Selenium-webdriver v4 & Chrome driver ^v.75)

Reply:
Hi @diemol 
Different ways resolve that issue. I made change again.

Reply:
I know, but the project is moving away from DesiredCapabilities to use browser options, such as `ChromeOptions` or `SafariOptions`, etc...